README.md

```diff
@@ -6,16 +6,22 @@ This package is distributed through [npm](https://www.npmjs.com/).
 
 ## File structure
 
-There are currently two directories, base and utilities. They are both intended to be used on any project but the key difference is how the code is used.
+There are currently three directories, config, variables and utilities. They are intended to be used on any project but the key difference is how the code is used.
 
-### Base
+### Variables
 
-Files in the base directory contain CSS which isn't targeted directly using a class but is compiled into the final CSS output (eg box-sizing or reset). It also contains usful mixins such as breakpoints.
+Files in variables contains specific variables for breakpoints, fonts, etc.
 
-### Utitilities
+### Config
+
+Files in the config directory contain CSS which isn't targeted directly using a class but is compiled into the final CSS output (eg box-sizing or reset). It also contains usful mixins such as breakpoints.
+
+### Utilities
 
 Files in utilities contains useful, compact snippets of code which can be accessed directly using a class in your HTML
 
+*Note* These files must be one of the last imported partials
```

Comment:
Individual utility files or the main `utilities.scss` must be one of the last imported partials

Reply:
ok done